core/lib/types/src/l2/mod.rs

```diff
@@ -233,7 +233,7 @@ impl L2Tx {
 
     /// Returns recipient account of the transaction.
     pub fn recipient_account(&self) -> Address {
```

Comment:
Shouldn't it be also `-> Option<Address>`?

Reply:
Indeed, updated!